platform/ios/src/MGLMapView.h

```diff
@@ -908,6 +953,24 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)setVisibleCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEdgeInsets)insets animated:(BOOL)animated;
 
+/**
+ Changes the receiver’s viewport to fit the given coordinate bounds and
+ optionally some additional padding on each side and a completion handler.
```

Comment:
This sentence sounds odd to me, perhaps consider rewriting it to something like:

>Changes the receiver’s viewport to fit the given coordinate bounds with optional padding on each side, and then triggering a completion handler.